### ISSUE TITLE
Missing right curly brace

### ISSUE THREAD

**Issue by SamHills** [AveYo/fox#6]
In the version I downloaded yesterday, there's a missing right curly brace.
`Line 7: $_Paste_in_Powershell = {` does not have a matching `}`.
I examined the code in both Powershell IDE and Notepad++, and neither can find the matching right curly brace for this line.

But today I looked again at the code posted on your GitHub site, and the problem has been fixed (a redundant `if` had been removed).

**Comment by AveYo** [comment]
well, last I've touched it was months ago. Keep up :)

**Comment by SamHills** [comment]
Earlier this week I clicked on the ChrEdgeFkOff link on rcmaehl's GitHub homepage [https://github.com/rcmaehl/MSEdgeRedirect](url) and it opened a GitHub page containing the code for ChrEdgeFkOff.  I saved this code to my HDD and studied it to see what it did.  I couldn't match up all the curly braces,  Powershell IDE and Notepad++ both showed the mismatch.  But somewhere during this time, some changes were made somewhere.   rcmaehl's GitHub page was updated 4 days ago and the ChrEdgeFkOff link now points to your homepage, not the ChrEdgeFkOff code page.  And the code on your GitHub site currently does not contain the extraneous `if` statement that was producing the curly brace mismatch.

A few minutes ago I searched for ChrEdgeFkOff on DuckDuckGo to see if that's where I had found the older version.  The first result it returned was:
> fox/ChrEdgeFkOff.cmd at main · AveYo/fox · GitHub
> [Search domain github.com] https://github.com › AveYo › fox › blob › main › ChrEdgeFkOff.cmd
> fox/ChrEdgeFkOff.cmd. This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository. #:: v2.0 only redirects microsoft-edge: links, no longer blocks msedge.exe (with a junction trick); v2.1 fix help urls.
Note the sentence beginning "This commit..."
It appears they indexed a bad version.
But when I clicked on the entry, it takes me to your current code.  Neither the bad `if` statement nor the comment about "This commit" are there.

Apparently there **was** an obsolete version out there somewhere but it no longer shows up anywhere and I can't find it again.
